packages/eslint-plugin-shopify/docs/rules/restrict-full-import.md

```diff
@@ -0,0 +1,38 @@
+# Prevents importing the entirety of a package. (restrict-full-import)
+
+Importing the entirety of a large module can be undesirable because it becomes harder to track what properties are being used.
+
+## Rule Details
+
+This rule aims to restrict the imports of specified modules to only be an import of specific properties. This does **not** mean that you are not importing from the module, it simply means that you only extract the properties you need.
```

Comment:
The first sentence isn't easy to read.  Why is it just aiming?  I'd prefer something like, "This rule prevents default imports for a configurable list of modules."

Second sentence's "does not mean that you are not" would be easier to parse without the double negative, e.g.,: "This is **not** a module blacklist; individual properties can still be imported."